test/jdk/ProblemList-FIPS140_3_OpenJcePlus.txt

```diff
@@ -426,12 +426,21 @@ sun/security/krb5/ktab/FileKeyTab.java https://github.com/ibmruntimes/openj9-ope
 sun/security/krb5/ktab/KeyTabIndex.java https://github.com/ibmruntimes/openj9-openjdk-jdk22/issues/68 linux-x64,linux-ppc64le,linux-s390x,aix-all
 sun/security/krb5/runNameEquals.sh https://github.com/ibmruntimes/openj9-openjdk-jdk22/issues/68 linux-x64,linux-ppc64le,linux-s390x,aix-all
 sun/security/krb5/auto/Cleaners.java https://github.com/ibmruntimes/openj9-openjdk-jdk22/issues/68 linux-x64,linux-ppc64le,linux-s390x
-java/lang/ProcessBuilder/JspawnhelperWarnings.java https://github.com/ibmruntimes/openj9-openjdk-jdk22/issues/68 linux-ppc64le,linux-s390x,aix-all
+java/lang/ProcessBuilder/JspawnhelperWarnings.java https://github.com/ibmruntimes/openj9-openjdk-jdk22/issues/68 linux-x64,linux-ppc64le,linux-s390x,aix-all
 java/lang/String/CompactString/NegativeSize.java https://github.com/ibmruntimes/openj9-openjdk-jdk22/issues/68 linux-ppc64le,linux-s390x,aix-all
-java/lang/Thread/virtual/stress/GetStackTraceALotWhenPinned.java#id0 https://github.com/ibmruntimes/openj9-openjdk-jdk22/issues/68 linux-ppc64le,linux-s390x,aix-all
+java/lang/Thread/virtual/stress/GetStackTraceALotWhenPinned.java#id0 https://github.com/ibmruntimes/openj9-openjdk-jdk22/issues/68 linux-x64,linux-ppc64le,linux-s390x,aix-all
 java/security/Security/ConfigFileTest.java https://github.com/ibmruntimes/openj9-openjdk-jdk22/issues/68 linux-ppc64le,linux-s390x
 javax/crypto/CipherSpi/ResetByteBuffer.java https://github.com/ibmruntimes/openj9-openjdk-jdk22/issues/68 linux-s390x
 java/security/Security/ConfigFileTest.java https://github.com/ibmruntimes/openj9-openjdk-jdk22/issues/68 linux-ppc64le,linux-s390x,aix-all
+java/lang/System/LoggerFinder/SignedLoggerFinderTest/SignedLoggerFinderTest.java https://github.com/ibmruntimes/openj9-openjdk-jdk22/issues/68 linux-x64,aix-all,linux-ppc64le,linux-s390x
+java/lang/Thread/virtual/ThreadAPI.java#default https://github.com/ibmruntimes/openj9-openjdk-jdk22/issues/68 linux-x64,aix-all,linux-ppc64le,linux-s390x
```

Comment:
Should we also add tests that are not FIPS140-3 specific in the exclude list? Don't these fail in regular runs too?

Reply:
FIPS problem list should only contain FIPS140-3 specific excludes. If this also fails in regular runs, it should not included here.

Reply:
updated

Reply:
@JinhangZhang please update jdk headstream exclude file as well, and backport to other streams.

Reply:
updating